dbconfig-common will use "sqlite3" driver. We map it to "sqlite". Closes: #714727

Index: roundcube/program/lib/Roundcube/rcube_db.php
===================================================================
--- roundcube.orig/program/lib/Roundcube/rcube_db.php	2015-03-22 20:09:41.057395516 +0100
+++ roundcube/program/lib/Roundcube/rcube_db.php	2015-03-22 20:09:41.050728784 +0100
@@ -65,6 +65,7 @@
         $driver     = strtolower(substr($db_dsnw, 0, strpos($db_dsnw, ':')));
         $driver_map = array(
             'sqlite2' => 'sqlite',
+            'sqlite3' => 'sqlite',
             'sybase'  => 'mssql',
             'dblib'   => 'mssql',
             'mysqli'  => 'mysql',
